tests/env.sh

```diff
@@ -1,7 +1,9 @@
-export BROKER_URL=localhost//guest:guest@rabbitmq//
-export DATABASE_URL=mysql://root@localhost:3306/treeherder
+export BROKER_URL=amqp://guest:guest@localhost:5672//
+export DATABASE_URL=${DATABASE_URL:-mysql://root@localhost:3306/treeherder}
 export REDIS_URL=redis://localhost:6379
 export SITE_URL=http://backend:8000/
 export TREEHERDER_DEBUG=True
 export TREEHERDER_DJANGO_SECRET_KEY=secret-key-of-at-least-50-characters-to-pass-check-deploy
 export NEW_RELIC_DEVELOPER_MODE=True
+
+exec "$@"
```

Comment:
This allows executing a command as part of initializing the env variables.

Reply:
This is standard scripting. In any case, we're hopefully getting rid of this file soon.

Reply:
`exec "$@"` is not needed. You are establishing a dependency in the code and you get no benefit. Consider the questions that arise from reading this line:

    ./tests/env.sh pytest tests

What does `env.sh` do with those parameters?  Are they directories?  Are they options?  This should be detailed in in `env.sh` description so the user can understand how to set those parameters.

Compare it to the equivalent:

    ./tests/env.sh 
    pytest tests

Which is clearer, because we state the side effects of `env.sh` and we assume the developer is familiar with `pytest` and its arguments.